protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -64,6 +64,11 @@ service TradingDataService {
     // Subscribe to a stream of Positions
     rpc ObservePositions(ObservePositionsRequest) returns (stream ObservePositionsResponse);
 
+    // LedgerEntries
```

Comment:
The idea of closeness (closing set of items under some operation) practically decides if we use `AND` or `OR` in the database query.
For transfers between accounts:
1->2,
2->3,
4->5,
4->7,
8->9
6->9
if we use account filters that match sending accounts: 1, 2, 4 and receiving accounts 5, 7 we have two options:
1. Close under From/To means display transfers:
4->5, 4->7 - meaning all transfers should match both AccountFilters (on From and To accounts) _at the same time_
=> example: `SELECT ... WHERE  (asset_id=$7 AND party_id=ANY($8) **AND** (market_id=ANY($9))`
2. Open under From/To means display transfers:
1->2, 2->3, 4->5, 4->7 - meaning all transfers that match AccountFilter settings in _at least_ one of the sides (From/To)
=> example `SELECT ... WHERE  (asset_id=$7 AND party_id=ANY($8) **OR** (market_id=ANY($9))`

Reply:
With the current implementation, after the corrections for all above comments, you can find all ledger entries for a party by selecting the party in the AccountFilter for `From` and `To` and use `CloseOnAccountFilters` = true; this will give you all ledger entries that have _both_ from and to accounts from that party.
If you want to also include transfers that this party made with _any_ other party - you set `CloseOnAccountFilters` to false.
If you want to check all transfers between party A and B - for example where sending party is A and receiving party is B - you set the party A from the `From` account AccountFilter and `B` party in the `To` account AccountFilter and set `CloseOnAccountFilters` to true => this will give you all transfers that have _only_ party A as sender and _only_ party B as receiver.
If you want all between A and B, regardless of the sending direction - you can set A and B in the `From` AccountFilter and  A and B in the `To` AccountFilter, set `CloseOnAccountFilters` to true and get all ledger entries where sending part is either A or B and receiving part is either A or B.
And that can go on.
The direction is (will be now after removing the IDs) set by the join dynamically.

But even though the results will be grouped by VegaTime, which means the results will not be individual in all cases.
So yes. 
I personally find it very confusing if one needs to find a single entry, or a small group of specific entries (I am not sure what we call small number of queries tbh) - they will be grouped by VegaTime in all cases first and adding 2-3 group settings (in order to reach them) will make the list large _and_ include a bunch of other stuff.
If we make the separation for aggregated and single - the aggregated one can use the current settings (with VegaTime) - like account filters (party, market, asset) + transfer types I think.
For the single entry finding method - we need to remove it I think. 

For the single (specific) entries - do we still need any grouping :thinking: